tests/platform_tests/test_first_time_boot_password_change/test_first_time_boot_password_change.py

```diff
@@ -0,0 +1,79 @@
+'''
+This test case checks default password change after initial reboot.
+Due to new law passed in California, each default user must change their default password.
+
+Important Note:
+    Please run this test from sonic-mgmt/tests folder, otherwise it will fail.
+'''
+import pexpect
+import time
+import pytest
+from tests.platform_tests.test_first_time_boot_password_change.default_consts import DefaultConsts
+from tests.platform_tests.test_first_time_boot_password_change.conftest import logger, currentConfigurations
+
+
+pytestmark = [
+    pytest.mark.topology('any'),
+    pytest.mark.sanity_check(skip_sanity=True),
+    pytest.mark.disable_loganalyzer,
+    pytest.mark.skip_check_dut_health
+]
+
+
+def test_default_password_change_after_first_boot(dut_hostname):
+    '''
+    @summary: in this test case we want to validate the mandatory request of
+    password change after the first boot of the given image.
+    According to a new law passed on the united states, default passwords
+    such as: "admin", "root", "12345", etc. are no longer accepted.
+    Test Flow:
+        1.A message should appear after initial boot, requesting password change for default user.
+        2.Password change, it will be tested by relogin to dut with new password and expecting no expire message again
+    :param dut_hostname: name of device under test
+    '''
+    logger.info("create ssh connection to device after inital boot")
+    engine = pexpect.spawn(DefaultConsts.SSH_COMMAND.format(DefaultConsts.DEFAULT_USER) + dut_hostname)
+    # to prevent race condition
+    engine.delaybeforesend = 0.2
+    engine.delayafterclose = 0.5
+    # it should require password so password will be sent
+    engine.expect(DefaultConsts.PASSWORD_REGEX)
```

Comment:
Suggest check if the change password feature enabled here, and if not enabled ignore this UT.

Because after this PR merged, in most case the feature will not enabled, then this UT may break some test pipeline.